README.md

```diff
@@ -674,41 +674,31 @@ And to seal the deal, let us show you how easily documents slot into your FastAP
 ```python
 import numpy as np
 from fastapi import FastAPI
-from httpx import AsyncClient
-
+from docarray.base_doc import DocArrayResponse
 from docarray import BaseDoc
 from docarray.documents import ImageDoc
 from docarray.typing import NdArray
-from docarray.base_doc import DocArrayResponse
-
 
 class InputDoc(BaseDoc):
     img: ImageDoc
+    text: str
 
 
 class OutputDoc(BaseDoc):
     embedding_clip: NdArray
     embedding_bert: NdArray
 
 
-input_doc = InputDoc(img=ImageDoc(tensor=np.zeros((3, 224, 224))))
-
 app = FastAPI()
 
+model_img, model_text = SomeEmbeddingModelImage(), SomeEmbeddingModelText()
```

Comment:
I prefer to have a `def embed_image` and `def embed_text` method with random numpy array generation, it is easier to reproduce